### DISCUSSION TITLE
Is it possible to get origin word and Superlative word from Comparative?

### DISCUSSION THREAD

**Discussion by ByungwooKim-BK** [nlp-compromise/nlp_compromise#1001] [Q&A] [answered]
If I request better, then I hope to get good and best from better.
Is it possible to get origin word and Superlative word from Comparative word?
Request 'Good' : 
Response
  Superlative: 'best',
  Comparative: 'better',
  Adverb: 'well',
  Noun: 'goodness',
  Verb: 'gooden'

Request 'Better'
Response
Superlative: 'betterest',
  Comparative: 'betterer',
  Adverb: 'betterly',
  Noun: 'betterness',
  Verb: null

Same way, hope to get origin word and comparative word from Superlative word.
Request 'Best'
Response
Superlative: 'bestest',
  Comparative: 'bester',
  Adverb: 'best',
  Noun: 'bestness',
  Verb: 'besten'

Do you have any good idea to get origin word?

**Comment by spencermountain** [comment, marked as answer]
hey @ByungwooKim-BK - yes, this is posible with `.adjectives().conjugate()` - please read the documentation [here](https://observablehq.com/@spencermountain/compromise-adjectives)
```js
console.log(nlp('better').adjectives().conjugate())
/*[
  {
    Adjective: 'good',
    Comparative: 'better',
    Superlative: 'best',
    Noun: 'goodness',
    Adverb: 'well'
  }
]
*/
```
cheers
